src/cli/conio.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * conio.cpp
+ * 
+ * Particle serial input and output for command line use
+ * 
+*/
+
+
+#include "conio.hpp"
+
+#include "Particle.h"
+
+#include <cstdio>
+
```

Comment:
You should probably explicitly include `stdarg.h`

Reply:
Done